Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Identity_Analytics.yml

```diff
@@ -0,0 +1,1821 @@
+      output:
+        simple: ${Account.Status}
+    - incidentfield: Cloud Account ID
+      output:
+        simple: ${MSGraphUser.ID}
+    - incidentfield: Account ID
+      output:
+        complex:
+          root: Account
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Account.Type
+                iscontext: true
+              right:
+                value:
+                  simple: Okta
+              ignorecase: true
+          - - operator: notContainsGeneral
+              left:
+                value:
+                  simple: Account.ID
+                iscontext: true
+              right:
+                value:
+                  simple: "="
+          accessor: ID
+    - incidentfield: Manager Email Address
+      output:
+        simple: ${UserManagerEmail}
+    - incidentfield: XDR Alert Search Results
+      output:
+        simple: ${PaloAltoNetworksXDR.Alert}
+    - incidentfield: Alert Name
+      output:
+        simple: ${inputs.AlertName}
+    - incidentfield: Detected User
+      output:
+        simple: ${inputs.Username}
+    - incidentfield: Username
+      output:
+        simple: ${inputs.Username}
+    - incidentfield: Number Of Found Related Alerts
+      output:
+        simple: ${NumOfRelatedAlerts}
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "12_13_#default#": 0.51,
+      "12_16_yes": 0.33,
+      "13_14_Non-Malicious": 0.61,
+      "13_16_Malicious": 0.35,
+      "22_23_yes": 0.44,
+      "22_24_#default#": 0.33,
+      "2_29_yes": 0.44,
+      "34_24_#default#": 0.31
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3235,
+        "width": 2320,
+        "x": -410,
+        "y": -240
+      }
+    }
+  }
+inputs:
+- key: AlertName
+  value: {}
+  required: false
+  description: Alert Name.
+  playbookInputQuery:
+- key: alert_id
+  value: {}
+  required: false
+  description: Alert ID.
+  playbookInputQuery:
+- key: IPAddress
+  value: {}
+  required: false
+  description: IP Address from the XDR Alert.
+  playbookInputQuery:
+- key: Username
+  value: {}
+  required: false
+  description: User name.
+  playbookInputQuery:
+- key: RelatedAlertsThreshold
+  value:
+    simple: "5"
+  required: false
+  description: |
+    This is the minimum threshold for XDR related alerts, based on MITRE tactics used to identify malicious activity by the user in the last 1 day.
+  playbookInputQuery:
+- key: FailedLogonThreshold
+  value:
+    simple: "30"
+  required: false
+  description: |-
+    This is the minimum threshold for user login failures within the last 1 day.
+    example: If this input is set to '30', and the 'Okta - User Investigation' or the 'Azure - User Investigation' sub-playbooks have found 31 failed login attempts - It will classify this behavior as malicious activity.
```

Comment:
```suggestion
    For example: If this input is set to '30', and the 'Okta - User Investigation' or the 'Azure - User Investigation' sub-playbooks have found 31 failed login attempts - It will classify this behavior as malicious activity.
```